numba/cuda/cudadrv/nvvm.py

```diff
@@ -195,6 +202,16 @@ def add_module(self, buffer):
                                                  len(buffer), None)
         self.driver.check_error(err, 'Failed to add module')
 
+    def lazy_add_module(self, buffer):
+        """
+        Lazily add an NVVM IR module to a compilation unit.
+        The buffer should contain NVVM module IR either in the bitcode
+        representation or in the text representation.
+        """
+        err = self.driver.nvvmLazyAddModuleToProgram(self._handle, buffer,
+                                                     len(buffer), None)
```

Comment:
Is there benefit in exposing the `name` parameter? May be for debug?

Reply:
I can't see any benefit in it - it was never exposed in `add_module` either - did you see a use for it?

Reply:
Was just a thought about debugging and tracking, it's not essential and trivial to add later, so think it's fine as-is.